metadata_backend/api/handlers.py

```diff
@@ -115,6 +116,23 @@ def get_page_param(param_name: str, default: int) -> int:
                             headers=link_headers,
                             content_type="application/json")
 
+    async def _get_data(self, req: Request) -> Dict:
+        """Get the data content from a request.
+
+        :param req: POST/PUT/PATCH request
+        :raises: HTTPBadRequest if request does not have proper JSON data
+        :returns: JSON content of the request
+        """
+        try:
+            content = await req.json()
+            # if content['type']
```

Comment:
probably remove this ?

Reply:
No idea what that was doing there, so a good catch 👍